src/horizon_api.ts

```diff
@@ -30,8 +30,8 @@ export namespace Horizon {
       > {
     created_at: string;
     fee_meta_xdr: string;
-    fee_charged: number;
-    max_fee: number;
+    fee_charged: number | string;
```

Comment:
would it make sense to represent it as a BigNumber?

Reply:
@tamirms so at the SDK level it comes as a string, consumers can then use BigNumber or any other library to parse it.